excitebike/src/test/java/com/codenjoy/dojo/excitebike/model/GameFieldImplSystemTest.java

```diff
@@ -30,6 +30,7 @@
 import com.codenjoy.dojo.services.EventListener;
 import com.codenjoy.dojo.utils.TestUtils;
 import org.junit.Before;
+import org.junit.Ignore;
```

Comment:
This import is redundant, could we remove it?

Reply:
yes